### ISSUE TITLE
[Accessibility][Screen reader]: Secondary text is not announced when user navigates in context menu under ContextualMenu with icons and secondary text [#6670]. (Bug 5)

### ISSUE THREAD

**Issue by jdhuntington** [OfficeDev/office-ui-fabric-react#10831]
Tracking all of the issues brought up with the 08/24 accessibility pass on Contextual Menu  control
Details on individual bugs can be found on [SharePoint](https://microsoft-my.sharepoint.com/:x:/r/personal/v-amaver_microsoft_com/_layouts/15/Doc.aspx?sourcedoc=%7Bcef6201c-bd85-44b6-a2d3-3784248461e0%7D&action=default&uid=%7BCEF6201C-BD85-44B6-A2D3-3784248461E0%7D&ListItemId=1259&ListId=%7B395E1ECB-5002-440E-81C5-6A090E98077C%7D&odsp=1&env=prodbubble)

Breakout of issue #6360

**Comment by joschect** [comment]
No repro: This appears to be fixed when I test the menu example.
